.eslintrc.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
I'd rather use yaml here. It's supported also by eslint and is a much richer syntax that allows i.e. comments.

Also you should disable any style-related checks from the file, and check only AST as explained in https://github.com/OCA/maintainer-quality-tools/pull/618#issuecomment-558576890.

You can draw inspiration from https://github.com/Tecnativa/oca-addons-repo-template/blob/99a27b5597a2bbfa6bd52f023556fdb188a4f567/.eslintrc.yml